shared/api/storage_pool.go

```diff
@@ -19,6 +19,10 @@ type StoragePool struct {
 	Name   string   `json:"name" yaml:"name"`
 	Driver string   `json:"driver" yaml:"driver"`
 	UsedBy []string `json:"used_by" yaml:"used_by"`
+
+	// API extension: clustering
+	State string   `json:"state" yaml:"state"`
```

Comment:
Should be Status (and "status" in API) for consistency

Reply:
This was fixed in 73d0351a1ce6716830848edb68499ef23b61c41c.